### ISSUE TITLE
MeshCommander can't enable IPv6, but it can enable from Intel AMT web console.

### ISSUE THREAD

**Issue by tedlee0502** [Ylianst/MeshCommander#5]
MeshCommander can't enable IPv6, but it can enable from Intel AMT web console.

![AMT_IPv6](https://user-images.githubusercontent.com/62271120/78766686-2af91180-79bc-11ea-88ca-a29bd0617924.jpg)

![AMT_IPv6_1](https://user-images.githubusercontent.com/62271120/78766699-30565c00-79bc-11ea-98d6-d2167f6d439e.jpg)

![MeshCommander_IPv6](https://user-images.githubusercontent.com/62271120/78766720-351b1000-79bc-11ea-8af5-53391257c538.jpg)

**Comment by Ylianst** [comment]
Yes, I think this feature worked on older Intel AMT, like 7.x. Not sure what is going on, will have to investigate this one.

**Comment by tedlee0502** [comment]
Good day. 
Can it be fixed at next version of MeshCommander ?
Can we know when the next version will be released ?

**Comment by zajdee** [comment]
I have just filled the same issue in #10, apologies. There are some traces there, so it might be useful after all.
